2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -1328,6 +1328,75 @@ Questo sprint impiega:
   supplement: "Grafico"
 )
 
+== Sprint 15 dal 11-02-2024 al 18-02-2024
+
+=== Obiettivi prefissati
+Questo sprint si occupa della preparazione per il colloquio RTB con il #vardanega e avvia le attività di progettazione del prodotto software. Gli obiettivi prefissati per questo sprint sono:
+- definizione del diagramma ER del database da implementare;
+- studio preliminare degli elementi architetturali del prodotto;
+- identificazione degli strumenti di appoggio per l'implementazione di una pipeline di CI/CD;
+- configurazione della repository e dei suddetti strumenti;
+- miglioramento della struttura informativa del documento #adr, in particolare per quanto riguarda il tracciamento tra casi d'uso e requisiti;
+- aggiornamento delle GitHub Actions recentemente deprecate;
```

Comment:
Quali? Li elencherei in modo che il contenuto del PdP sia quanto più informativo possibile